.rubocop.yml

```diff
@@ -10,15 +10,31 @@ AllCops:
 Gemspec/RequireMFA:
   Enabled: false
 
+Metrics/AbcSize:
+  Exclude:
+    - 'gems/smithy-client/lib/smithy-client/cbor/encoder.rb'
+    - 'gems/smithy-client/lib/smithy-client/cbor/decoder.rb'
+    - 'gems/smithy-client/spec/smithy-client/cbor_spec.rb'
+
 Metrics/BlockLength:
   Exclude:
     - '**/spec/**/*.rb'
 
 Metrics/ClassLength:
   Max: 200
+  Exclude:
+    - 'gems/smithy-client/lib/smithy-client/cbor/decoder.rb'
+
+Metrics/CyclomaticComplexity:
+  Exclude:
+    - 'gems/smithy-client/lib/smithy-client/cbor/encoder.rb'
+    - 'gems/smithy-client/lib/smithy-client/cbor/decoder.rb'
+    - 'gems/smithy-client/spec/smithy-client/cbor_spec.rb'
 
 Metrics/MethodLength:
   Max: 20
+  Exclude:
+    - 'gems/smithy-client/spec/smithy-client/cbor_spec.rb'
```

Comment:
Specs should already be excluded? If not you should do this generally (see BlockLength)